global.json

```diff
@@ -461,7 +461,7 @@
         "id": "wfm_project",
         "priority": 0.3,
         "name": "WFM Project",
-        "description": "FeedHenry WFM project",
+        "description": "FeedHenry WFM project. Please follow WFM documentation before creating new project.",
```

Comment:
Do we need to add a link to the wfm docs here?

Reply:
We do not know the link yet until it will be released. Having the link that will not work properly after it's too much of a risk.